src/modules/workflows/enums.js

```diff
@@ -0,0 +1,19 @@
+const CreateProcess = Object.freeze({
```

Comment:
Why not a constants folder common to the whole repo ?
That would be easier to trace and maintain.

Reply:
there can be multiple constants across modules, better keep individual to modules,  if needed we can import from respective modules.